idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/RemoveModifierFix.kt

```diff
@@ -51,6 +51,9 @@ class RemoveModifierFix(
     }
 
     companion object {
+        val removeRedundantModifier = createRemoveModifierFactory(isRedundant = true)
```

Comment:
Super minor: I couldn't find a definitive answer on this, but the factories in `ChangeVariableMutabilityFix.Companion` have `UPPER_CASE_NAMES`. Which one is the correct naming convention? 🙂  https://kotlinlang.org/docs/coding-conventions.html#property-names seems to indicate it should be `removeRedundantModifier` since the property value has behavior.

Reply:
Unfortunately, we do not have a specific code-style convention in Kotlin project for that. Personally, I prefer naming immutable static properties like factories or applicators as UPPER_CASE, but do not see any problems with camelCase here